src/main/java/org/terasology/launcher/LauncherInitTask.java

```diff
@@ -98,11 +95,11 @@ protected LauncherConfiguration call() {
 
             updateMessage(I18N.getLabel("splash_fetchReleases"));
             logger.info("Fetching game releases ...");
+            // implicitly fetches game releases and cache them
             final RepositoryManager repositoryManager = new RepositoryManager(client);
-            Set<GameRelease> releases = repositoryManager.getReleases();
```

Comment:
do we call `getReleases()` in other situations or can it be removed?

Reply:
Yes, in the application controller when we assemble the list of releases the user can choose from (where we actually need them):

https://github.com/MovingBlocks/TerasologyLauncher/blob/f5998e3168fe77dc78a5ee4b3e0077f313f219d0/src/main/java/org/terasology/launcher/ui/ApplicationController.java#L205-L216